AdblockPlusSafari/AdblockPlus.h

```diff
@@ -17,22 +17,28 @@
 
 #import <Foundation/Foundation.h>
 
+extern NSString *_Nonnull AdblockPlusActivated;
+
 @interface AdblockPlus : NSObject
 
+@property (nonatomic, strong, readonly) NSUserDefaults *__nonnull adblockPlusDetails;
+
+- (NSString *__nonnull)group;
+
+- (NSString *__nonnull)contentBlockerIdentifier;
+
+- (NSString *__nonnull)backgroundSessionConfigurationIdentifier;
+
 @property (nonatomic) BOOL enabled;
 
 @property (nonatomic) BOOL acceptableAdsEnabled;
 
 @property (nonatomic) BOOL activated;
 
-@property (nonatomic) BOOL reloading;
-
-- (void)setEnabled:(BOOL)enabled reload:(BOOL)reload;
-
-- (void)setAcceptableAdsEnabled:(BOOL)enabled reload:(BOOL)reload;
+@property (nonatomic) NSInteger installedVersion;
```

Comment:
I talked to Pavel about it and we decided to stay with version. The reason behind this is some case dirty flag might be overridden with newest update filter. Then it would take 5 days to update filter.